### PR TITLE
Add job listings

### PR DESCRIPTION
**Pull request by MarkBennett**
This is an MVP for job listings.

The jobs themselves appear on the homepage with a link to a more detailed page for each listing.

TODO:
* [x] Add a single page dedicated to job listings
* [x] Add a link to submit a job

### CODE REVIEW

**Comment by MarkBennett** [comment]
@CAYdenberg any change of a code review?

File: views/job.hbs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <h3>Summary</h3>
+        {{{ summary.html }}}
+
+        <h3>Work Description</h3>
+        {{{ work_desc.html }}}
+
+        <h3>Requirements</h3>
+        {{{ requirements.html }}}
+
+        <h3>Why You Should Apply</h3>
```
Comment (CAYdenberg) [inline views/job.hbs]: This section seems non-standard for job descriptions. Only concern that admin (ie you or me) could end up doing work to create it? Maybe should be optional (`{{# if}}`)

File: views/index.hbs [hunk 2]
```diff
@@ -14,7 +14,9 @@
       <p>
         Talks range from beginner to advanced and include the full stack of popular JavaScript
         technologies from jQuery to CoffeeScript to Node.js. It is also common to hear topics covering
-        wider Web-technology trends and evolving standards.
+        wider Web-technology trends and evolving standards. <a href="https://docs.google.com/spreadsheets/d/1_FOG1K_fUbTlNvqT9hjihiXYt9AhPkZxW6tfZN858VU/edit?usp=sharing">Sign up to talk</a>, or
+        <a href="https://github.com/ExchangeJS/exchangejs-org/issues/new?title=(YOUR%20TALK%20NAME%20HERE)&body=What,%20why,%20who?&labels=talk-ideas">
```
Comment (MarkBennett) [inline views/index.hbs]: Good call.

File: views/job.hbs (hunk 1 above)
Comment (MarkBennett) [inline views/job.hbs]: TBH, I just copied it from the LinkedIn job listings that Jobber used. I'll make it optional like you suggest but I have a feeling the structure of these job listings will probably get tweaked as we get feedback.

**Comment by MarkBennett** [comment]
You good with me merging the latest @CAYdenberg?

**Comment by CAYdenberg** [comment]
:rocket:

**Pull request merged**